.meta/sinks/aws_s3.toml.erb

```diff
@@ -69,7 +69,8 @@ description = "The S3 bucket name. Do not include a leading `s3://` or a trailin
 
 <%= render("_partials/fields/_encoding_options.toml",
   namespace: "sinks.aws_s3.options",
-  encodings: ["ndjson", "text"]
+  encodings: ["ndjson", "text"],
+  default: "text"
```

Comment:
Was this an intentional change? Same with the `kafka` and `splunk_hec` files below.

Reply:
Yes, I noticed that `pulsar` need to be fixed, but also fixed 3 another sinks in the same commit because thought that these trivial changes and not require separate PR. Happy to move them if this required.

Reply:
That's fine, just making sure!